frontend/packages/dev-console/src/components/project-access/__tests__/project-access-form-utils.spec.ts

```diff
@@ -18,4 +23,23 @@ describe('Fetch required roles', () => {
     const userRoleBindings = getUserRoleBindings(roleBindingsWithRequiredRoles);
     expect(userRoleBindings).toEqual(roleBindingsWithRequiredAttributes);
   });
+
+  it('should ignore the role binding name from the role binding obj', async () => {
+    const roleBinding: UserRoleBinding[] = [
+      {
+        user: 'ab',
+        role: 'edit',
+      },
+      { user: 'kubeadmin', role: 'admin' },
+      { user: 'de', role: 'view' },
+    ];
```

Comment:
@debsmita1 here in the mock roleBindings you havent added roleBindingName in any obj so `ignoreRoleBindingName` is not getting tested fully IMO. The array is simply getting sorted which is not the only thing this function does.

Reply:
Fixed